specification/consumption/resource-manager/readme.md

```diff
@@ -190,6 +191,14 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 output-folder: $(go-sdk-folder)/services/consumption/mgmt/2018-01-31/consumption
 ```
 
+### Tag: package-2018-03 and go
```

Comment:
Depending on your response to my comment:

> Is there still a January version of this, or did shipping schedule change?

The `package-2018-01 and go` clause may need to be deleted.

Reply:
Good catch! It was just overwritten. I have added the January version back.